lib/receivers/tcpReceiver.js

```diff
@@ -12,6 +12,25 @@
 
 const net = require('net');
 const log = require('../log/logger');
+const { serialize } = require('../utils');
```

Comment:
Just stylistic here, does destructuring make sense since we're only assigning to a single variable?

Reply:
I believe this is enforced by the airbnb style guide config for eslint (at least the eslint in the JS SDK was enforcing it, and I believe it wasn't because it was a custom rule ontop of the airbnb guide). I'm used to it, but not particularly attached to it. I think it's fine either way. Shall I change it?

Reply:
It's fine...  We don't enforce it as part of our style guide, but I'm not completely opposed to it...